src/analysis/Environments/Environments.test.ts

```diff
@@ -804,6 +804,57 @@ describe('Environments', () => {
       screen.getByText('Delete persistent disk?');
     });
   });
+
+  describe('Other - ', () => {
+    it('Performs query properly with resources owner checkbox', async () => {
```

Comment:
the environment page is an early adopter of the providers, this is how simple our tests will read once we finish AOU code sharing efforts!!